target_chains/ethereum/examples/coin_flip/app/src/flip_coin.ts

```diff
@@ -106,7 +106,7 @@ async function main() {
   // Note that there is a potential race condition here: the server may not have observed the request ^
   // before this HTTP response. Hence, we retry fetching the url a couple of times.
   const response = await fetchWithRetry(url, 3);
-  const providerRandom = web3.utils.bytesToHex(response.value);
+  const providerRandom = `0x${response.value.data}`;
```

Comment:
adjust example since this PR changed the API https://github.com/pyth-network/pyth-crosschain/pull/1129